modules/survey_module/jsx/DirectEntry.js

```diff
@@ -0,0 +1,384 @@
+/**
+ * This file contains React component for Direct Data Entry
+ *
+ * @author Jordan Stirling (StiringApps ltd.)
```

Comment:
I'm not sure about advertising in the codebase... but in any case you've got a typo here 😸